examples/player/simid_player.js

```diff
@@ -430,11 +525,14 @@ class SimidPlayer {
   
   /** The creative wants to play video. */
   onRequestPlay(incomingMessage) {
-    this.adVideoElement_.play().then(
-      // The play function returns a promise.
-      this.simidProtocol.resolve(incomingMessage),
-      this.simidProtocol.reject(incomingMessage)
-    );
+    
```

Comment:
I think I might have already commented this on a different cl (maybe duplicating some commits?), but I think these should be `() => this.simidProtocol.resolve(message)`

Reply:
kaleab fixed